rest-api-spec/src/main/resources/rest-api-spec/api/ingest.put_pipeline.json

```diff
@@ -27,6 +27,10 @@
       ]
     },
     "params":{
+      "if_version":{
+        "type":"string",
```

Comment:
Should be `type: int` instead of `string`

Reply:
This was intentional. See [this comment](https://github.com/elastic/elasticsearch/pull/78551#issuecomment-932441576) above about handling existing pipelines with null-valued `version` attributes.